packages/lib-react-components/src/ZooHeader/ZooHeader.js

```diff
@@ -79,12 +78,12 @@ export default function ZooHeader (props) {
       {...props}
     >
       <Box
+        as='nav'
```

Comment:
[minor] Huh, I didn't notice that some of our components were still using the [deprecated `tag` property.](https://v2.grommet.io/box#tag)